setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+import glob
+
+templates = glob.glob("templates/*")
+model_spec = glob.glob("acis_thermal_check/*_check/*.json")
+
+entry_points = {}
+entry_points['console_scripts'] = [
+    'dea_check = acis_thermal_check.dea_check.dea_check:main',
+    'dpa_check = acis_thermal_check.dpa_check.dpa_check:main',
+    'psmc_check = acis_thermal_check.psmc_check.psmc_check:main',
+]
+
+
+setup(name='acis_thermal_check',
+      packages=find_packages(),
+      version='1.0.0',
```

Comment:
Doing something like `from acis_thermal_check import __version__` and then referencing that `__version__` seems to work well so the version is only coded in one place instead of 3.  It does require being able to import the package before building, but in this case it's not a problem.